CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+-   Resolve Avatar size shortcode attribute issue in donor wall shortcode and adds support for avatar size in donor wall block (#5443)
```

Comment:
Please move this into the `Unreleased` section

Reply:
This has been moved

Reply:
@DevinWalker thank you for moving that. I misplace when resolving conflict.

Reply:
@DevinWalker I noticed that it was not placed under a section (`Added, Fixed, etc.`). I moved it to the `Unreleased-->Added` section.